Lavinia-api/Models/Country.cs

```diff
@@ -4,6 +4,7 @@
 
 namespace LaviniaApi.Models
 {
+    // API v1
```

Comment:
Overall I think we should have used namespaces to distinguish between the APIs, but that isn't a big concern at the moment, and we'll probably deprecate API v1 entirely after API v2 is ready for production.